packages/loading-scripts/src/index.ts

```diff
@@ -12,6 +12,8 @@ import type {
   RudderAnalyticsPreloader,
 } from '@rudderstack/analytics-js';
 
+const snippetVersion = '__PACKAGE_VERSION__';
```

Comment:
It'd be great if this snippet version were set as a property of the analytics object itself. Might be helpful to support multiple snippet versions etc.
Anyway, it's not super urgent.

Reply:
why whould we need multible snippet versions?

Reply:
Recently, we realized a bug and updated our snippet. It might be possible that newer snippet versions may not be backward compatible with older SDKs versions. I understand that in these situations, the SDK would've also leaped forward in the major version.

However, the SDK knowing the snippet version might help tune itself accordingly if needed. I'm not saying there will be a need like this, but it is also helpful for other things like the snippet version can be included in the reporting data, etc.

Reply:
Hm hm..... would you see this information more usefull in event library object or in bugnag meta?

Reply:
Actually, both as in Bugsnag we don't have access to the event data itself.

Reply:
Added in both